src/Analytics/Renderer/Google/GoogleClassicAnalytics.php

```diff
@@ -52,7 +52,12 @@ public function getScript()
         $script[] = '<script>';
 
         foreach ($this->participations as $testIdentifier => $variationIndex) {
-            $script[] = "cxApi.setChosenVariation(" . (int) $variationIndex . ", '" . (string) $testIdentifier . "')";
+            $script[] = "cxApi.setChosenVariation(" . (int) $variationIndex . ", '" . (string) $testIdentifier . "');";
+        }
+
+        if (true === $this->getEventTriggerInclusion()) {
+            $script[] = "_gaq.push(['_trackEvent', 'PhpAb', 'testRun', 'testsAmout', "
```

Comment:
True. It happens that we might need to send an event per test which would make it like:
```
"_gaq.push(['_trackEvent', 'PhpAb', 'testIdentifier', 'variantIdentifier']);
```